eu.quelltext.open_chakra_toning.yml

```diff
@@ -0,0 +1,44 @@
+# Open Chakra Toning
+# Copyright (C) 2024  Nicco Kunzmann
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+id: eu.quelltext.open_chakra_toning
```

Comment:
How do we verify quelltext.eu belongs to the project? It redirects to https://hosted.quelltext.eu/login, maybe put some thing up?

Reply:
```
$ ping -4 quelltext.eu
PING  (78.47.87.181) 56(84) bytes of data.
```

![image](https://github.com/flathub/flathub/assets/564768/1b9e0d3a-8137-42b2-bc7f-526ee2f0b4b3)

Reply:
I can also move the hosting of the Github pages over.
I did not know that domain verification is required.

Reply:
I don't know what the screenshot is supposed to show. We need to check that you or the project owns the domain used and it's not someone else's. That's what is meant by the app author controls the domain.

The way to do that would be to link the repo or make a page about the app on the domain.

Pinging doesn't prove anything, I can ping example.org, doesn't mean I own the domain.

Reply:
>  I did not know that domain verification is required.

You acknowledged:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

But without a way to simply verify that... 

People put a webpage for the software, some link to what they do, etc. So it's often easy to reconcile the author with the domain. If it is parked mostly it's impossible to verify.

Reply:
Yeah, I see that. Thanks... I did not know that. E.g. when I submitted to F-Droid, they were file with the unique id I used for my apps. We can just wait and I will host it there, then :) Thanks for taking the time!

Reply:
http://open-chakra-toning.quelltext.eu/ shows the app now. I think, this should resolve the issue.